object_service/tests/test_queries.py

```diff
@@ -33,7 +34,7 @@ def test_query_parsing(self):
             'citations(reviews(popular(object:("M 1" OR M81) OR =abs:Andromeda))) year:2010 property:refereed': [['M 1', 'M81'],['object:("M 1" OR M81)']],
             }
 
-        for qstring, expected in test_cases.items():
+        for qstring, expected in list(test_cases.items()):
```

Comment:
I think this `list(...)` (automatically added by the porting tools) might not be needed here.